networks/suzuka/suzuka-full-node/src/tasks/execute_settle.rs

```diff
@@ -130,17 +131,11 @@ where
 			anyhow::bail!("Invalid DA height: {:?}", da_height);
 		}
 
-		// decompress the block bytes
-		let block = tokio::task::spawn_blocking(move || {
-			let decompressed_block_bytes = zstd::decode_all(&block_bytes[..])?;
-			let block: Block = bcs::from_bytes(&decompressed_block_bytes)?;
-			Ok::<Block, anyhow::Error>(block)
-		})
-		.await??;
+		let block: Block = bcs::from_bytes(&block_bytes[..])?;
 
 		// get the transactions
 		let transactions_count = block.transactions().len();
-		let span = info_span!(target: "movement_timing", "execute_block", id = %block_id);
+		let span = info_span!(target: "movement_timing", "execute_block", id = ?block_id);
```

Comment:
The debug representation of `Vec` is going to be a weird microformat in telemetry (which #697 will turn this span into); do a hex here as well?

Reply:
Sure, I can do a hex.